src/pages/SpacesDashboard/SpacesDashboard.tsx

```diff
@@ -30,26 +32,23 @@ import { TesterRestricted } from "components/atoms/TesterRestricted";
 import "./SpacesDashboard.scss";
 
 interface SpacesDashboardProps {
-  ownSpaces: Spaces;
   userId: UserId;
   world: WorldWithId;
   worldId: WorldId;
   worldSlug: WorldSlug;
 }
 
 export const SpacesDashboard: React.FC<SpacesDashboardProps> = ({
-  ownSpaces,
   userId,
   world,
   worldId,
   worldSlug,
 }) => {
   const isWorldAdmin = userId ? world?.owners.includes(userId) : undefined;
 
-  const spaces = useMemo(
-    () => (world ? ownSpaces.filter((venue) => venue.worldId === worldId) : []),
-    [ownSpaces, world, worldId]
-  );
+  const { spaces } = useWorldSpaces({ worldId });
+
+  console.log({ worldId, spaces });
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/2820/commits/af825f995a7267c0da1cdbc23ea8630055576519